classes/Http/Form/ResetForm.php

```diff
@@ -34,7 +34,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->getForm();
     }
 
-    public function getName()
+    public function getName(): string
```

Comment:
Not even sure whether this method is used by anything.

Reply:
I can't find any place that uses it. Looks safe to delete.